ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -631,6 +642,31 @@ def run_ocs_upgrade(operation=None, *operation_args, **operation_kwargs):
             time.sleep(30)
             # End of workaround
 
+        # update external secrets
+        if config.DEPLOYMENT["external_mode"] and not (
+            original_ocs_version == "4.7" and upgrade_version == "4.8"
+        ):
+            external_cluster.update_permission_caps()
```

Comment:
Does this require the bug with caps to be backported to 4.8? Or it's something different here?

I remember that last conversation was not finalized to which version the bug will be backported. @agarwal-mudit might know more, but on last PC I remember he told that we will need another meeting to decide.

Reply:
Yes, we need to backport it to 4.9 and 4.8
@parth-gr can add more.

Reply:
Ohh is the automatic upgrade we have for external cluster?

No, we don't need any backport in case upgrade.

I want to understand what the change is about ig the things are already present you are moving parts more modular, Or I will say please update the PR description.

Reply:
this PR is to create and run the script dynamically ( previously we have harcoded values for keys where user send as input to automation ) and also added few missing steps post upgrade.

Reply:
Thanks for the change, 
I want to discuss with you the points for 4.10, how we should upgrade in that scenario because the upgrade flag we have used till 4.9 is updated in 4.10 script.

Reply:
We can take this offline referr:https://github.com/rook/rook/blob/a585a69843fff3a5731ec0c21f02cbca37f4e687/deploy/examples/create-external-cluster-resources.py#L220